config/environments/development.rb

```diff
@@ -90,8 +91,6 @@
       end
     end
 
-    ActiveRecord::Base.logger = nil if ENV["RAILS_DISABLE_ACTIVERECORD_LOGS"] == "1"
```

Comment:
Not sure why this was setting `ActiveRecord::Base` directly in the `after_initialize` block when there is `config.active_record.logger`